src/Database/Dialect/PostgresDialectTrait.php

```diff
@@ -186,4 +186,12 @@ public function enableForeignKeySQL()
     {
         return 'SET CONSTRAINTS ALL IMMEDIATE';
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function explainSQL($query)
```

Comment:
I am file to `array $options = []` as the second argument. but it would be difficult to achieve as abstraction. There are many non-compatible options each driver.